backend/apis/leads/list-leads.api.js

```diff
@@ -29,6 +29,14 @@ app.get("/api/leads", (req, res, next) => {
     );
   }
 
+  let whereClause = `WHERE leads.isDeleted = false`;
+  let whereClauseValues = [];
+
+  if (req.query.name) {
+    whereClause += `AND CONCAT(leads.firstName, ' ', leads.lastName) LIKE ? `;
```

Comment:
don't we need a space character at the beginning of the string here, so that `false` on line 32 doesn't mash up against `AND` on line 36?